execution/executor/src/block_tree/block_tree_test.rs

```diff
@@ -32,14 +32,14 @@ impl Block for TestBlock {
         self.output.is_some()
     }
 
-    fn set_output(&mut self, output: Self::Output) {
+    fn set_output(&mut self, _output: Self::Output) {
```

Comment:
This is a semantic change. I'm surprised to see it bundled with the rest of this PR and would like to see some commentary on why it is needed. In particular, I'm concerned that there isn't any tests that care about out this being the one way or the other.

Reply:
I was concerned as well, but if you look higher up you can see that the associated type here is () and this makes the function match the other implementations which also return the type directly instead of forwarding the passed in value. So for that reason I approved it.

Reply:
This is the analysis that prompted cargo-fix to make this change : 
https://github.com/rust-lang/rust/blob/f789b6bd6dc01a4412819c0733a6c99e092446fe/src/librustc/middle/liveness.rs#L1546-L1568

Reply:
I see now. It would have been much easier to understand in situ if the signature used () instead of Self::Output.